libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1561,9 +1561,10 @@ bool PbftManager::pushCertVotedPbftBlockIntoChain_(taraxa::blk_hash_t const &cer
     return false;
   }
 
-  cert_sync_block_.cert_votes = std::move(cert_votes_for_round);
+  // TODO: Only include reward votes that are part of the pbft block and not all cert votes
+  cert_sync_block_.cert_votes = db_->getLastBlockCertVotes();
```

Comment:
do we need to get them from DB? isn't it better to get them from some memory and this should be used on restart?

Reply:
That will change in my PR, get them from reward votes in memory